csharp/ql/test/library-tests/linq/Linq1.expected

```diff
@@ -26,6 +26,8 @@
 | queries.cs:27:11:27:42 | call to method Join | 2 | queries.cs:27:21:27:25 | access to local variable list1 |
 | queries.cs:27:11:27:42 | call to method Join | 3 | queries.cs:27:30:27:33 | access to local variable next |
 | queries.cs:27:11:27:42 | call to method Join | 4 | queries.cs:27:42:27:42 | access to local variable c |
+| queries.cs:28:11:28:18 | call to method  | 0 | queries.cs:27:11:27:42 | call to method Join |
```

Comment:
Why not `call to method Join` ?

Reply:
Actually, it is a call to `Select` that is missing. It appears to be a Roslyn bug: [this](https://github.com/Semmle/ql/blob/master/csharp/extractor/Semmle.Extraction.CSharp/Entities/Expressions/Query.cs#L259) call always return `null` for LINQ expressions containing `join`s.